docs/configuration-parameters.md

```diff
@@ -208,6 +208,7 @@ RESET
 | carbon.options.date.format                 | Specifies the data format of the date columns in the data being loaded |
 | carbon.options.timestamp.format            | Specifies the timestamp format of the time stamp columns in the data being loaded |
 | carbon.options.sort.scope                 | Specifies how the current data load should be sorted with. **NOTE:** Refer to [Data Loading Configuration](#data-loading-configuration)#carbon.sort.scope for detailed information. |
+| carbon.table.load.sort.scope              | Overrides the SORT_SCOPE provides in CREATE TABLE.                         |
```

Comment:
```suggestion
| carbon.table.load.sort.scope              | Override the SORT_SCOPE provided in CREATE TABLE.                         |
```

Reply:
"provides" changed to "provided". This was a spelling mistake.
"Overrides" is correct. No Change.